src/nni_manager/training_service/kubernetes/kubeflow/kubeflowData.ts

```diff
@@ -0,0 +1,39 @@
+/**
```

Comment:
most of content in kubeflowData.ts and generateRunScript() method are duplicated with frameworkcontroller's corresponding logic. I STRONGLY suggest to remove such duplication, for clear code and better maintenance.